### PR TITLE
small(?) changes to team technical standards

### PR DESCRIPTION
**Pull request by afeld**
Easiest to [review ignoring whitespace](https://github.com/18F/tts-tech-portfolio/pull/1476/files?diff=unified&w=1). Added comments by the substantive changes.

### CODE REVIEW

**Comment by JJediny** [comment]
Terraform plan in __terraform__
<details>
  <summary>Plan: 36 to add, 0 to change, 0 to destroy.</summary>

```hcl
Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # local_file.github_repos will be created
  + resource "local_file" "github_repos" {
      + content              = <<-EOT
            _This file is generated by [Terraform](../terraform/github.md.tmpl)._
            
            # GitHub Repos
            
            | repo | status | Tech Portfolio manages all content? |
            | --- | --- | --- |
            | [18F/.github](https://github.com/18F/.github) | active | N |
            | [18F/aws-admin](https://github.com/18F/aws-admin) | active | Y |
            | [18F/aws-admin-cleanup](https://github.com/18F/aws-admin-cleanup) | active | Y |
            | [18F/before-you-ship](https://github.com/18F/before-you-ship) | archived | Y |
            | [18F/billing-tools](https://github.com/18F/billing-tools) | archived | Y |
            | [18F/bug-bounty](https://github.com/18F/bug-bounty) | active | Y |
            | [18F/certificate-service](https://github.com/18F/certificate-service) | archived | Y |
            | [18F/chandika](https://github.com/18F/chandika) | archived | Y |
            | [18F/charlie](https://github.com/18F/charlie) | active | Y |
            | [18F/chat](https://github.com/18F/chat) | active | Y |
            | [18F/deploy-ttslicenses](https://github.com/18F/deploy-ttslicenses) | archived | Y |
            | [18F/dns](https://github.com/18F/dns) | active | N |
            | [18F/ghad](https://github.com/18F/ghad) | active | Y |
            | [18F/handbook](https://github.com/18F/handbook) | active | N |
            | [18F/join.tts.gsa.gov](https://github.com/18F/join.tts.gsa.gov) | active | N |
            | [18F/laptop](https://github.com/18F/laptop) | archived | Y |
            | [18F/newrelic-terraform](https://github.com/18F/newrelic-terraform) | active | Y |
            | [18F/pages-redirects](https://github.com/18F/pages-redirects) | active | Y |
            | [18F/private-eye](https://github.com/18F/private-eye) | active | Y |
            | [18F/raktabija](https://github.com/18F/raktabija) | archived | Y |
            | [18F/slack-export-handling](https://github.com/18F/slack-export-handling) | archived | Y |
            | [18F/tts-common-controls](https://github.com/18F/tts-common-controls) | active | Y |
            | [18F/tts-tech-portfolio](https://github.com/18F/tts-tech-portfolio) | active | Y |
            | [18F/tts-tech-portfolio-private](https://github.com/18F/tts-tech-portfolio-private) | active | Y |
            | [18F/uswds-jekyll](https://github.com/18F/uswds-jekyll) | active | Y |
            | [18F/vulnerability-disclosure-policy](https://github.com/18F/vulnerability-disclosure-policy) | active | Y |
            
            For repositories where the Tech Portfolio _doesn't_ manage all content, we are the "technical owners", meaning we keep them running, fix any security vulnerabilities, etc. In the Handbook, we are responsible for pages related to things we own, like those under [software and tools](https://handbook.tts.gsa.gov/#software-and-tools).
            
            ## Adding a repository
            
            When a repository is considered "managed" by the Tech Portfolio, it should be included in the following places:
            
            - [The list in Terraform](../terraform/main.tf)
            - Linked repositories for project boards
              - [Team board](https://github.com/orgs/18F/projects/11/settings/linked_repositories)
              - [Pull requests](https://github.com/orgs/18F/projects/19/settings/linked_repositories)
        EOT
      + directory_permission = "0755"
      + file_permission      = "0644"
      + filename             = "./../how_we_work/github.md"
      + id                   = (known after apply)
    }

  # module.cloud_gov.github_membership.owner["JJediny"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "JJediny"
    }

  # module.cloud_gov.github_membership.owner["adborden"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "adborden"
    }

  # module.cloud_gov.github_membership.owner["afeld"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "afeld"
    }

  # module.cloud_gov.github_membership.owner["rocheller123"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "rocheller123"
    }

  # module.cloud_gov.github_membership.owner["tts-bot"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "tts-bot"
    }

  # module.digital_analytics_program.github_membership.owner["JJediny"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "JJediny"
    }

  # module.digital_analytics_program.github_membership.owner["adborden"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "adborden"
    }

  # module.digital_analytics_program.github_membership.owner["afeld"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "afeld"
    }

  # module.digital_analytics_program.github_membership.owner["rocheller123"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "rocheller123"
    }

  # module.digital_analytics_program.github_membership.owner["tts-bot"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "tts-bot"
    }

  # module.federalist_users.github_membership.owner["JJediny"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "JJediny"
    }

  # module.federalist_users.github_membership.owner["adborden"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "adborden"
    }

  # module.federalist_users.github_membership.owner["afeld"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "afeld"
    }

  # module.federalist_users.github_membership.owner["rocheller123"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "rocheller123"
    }

  # module.federalist_users.github_membership.owner["tts-bot"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "tts-bot"
    }

  # module.fellows_in_innovation.github_membership.owner["JJediny"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "JJediny"
    }

  # module.fellows_in_innovation.github_membership.owner["adborden"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "adborden"
    }

  # module.fellows_in_innovation.github_membership.owner["afeld"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "afeld"
    }

  # module.fellows_in_innovation.github_membership.owner["rocheller123"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "rocheller123"
    }

  # module.fellows_in_innovation.github_membership.owner["tts-bot"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "tts-bot"
    }

  # module.presidential_innovation_fellows.github_membership.owner["JJediny"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "JJediny"
    }

  # module.presidential_innovation_fellows.github_membership.owner["adborden"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "adborden"
    }

  # module.presidential_innovation_fellows.github_membership.owner["afeld"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "afeld"
    }

  # module.presidential_innovation_fellows.github_membership.owner["rocheller123"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "rocheller123"
    }

  # module.presidential_innovation_fellows.github_membership.owner["tts-bot"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "tts-bot"
    }

  # module.project_open_data.github_membership.owner["JJediny"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "JJediny"
    }

  # module.project_open_data.github_membership.owner["adborden"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "adborden"
    }

  # module.project_open_data.github_membership.owner["afeld"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "afeld"
    }

  # module.project_open_data.github_membership.owner["rocheller123"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "rocheller123"
    }

  # module.project_open_data.github_membership.owner["tts-bot"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "tts-bot"
    }

  # module.uswds.github_membership.owner["JJediny"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "JJediny"
    }

  # module.uswds.github_membership.owner["adborden"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "adborden"
    }

  # module.uswds.github_membership.owner["afeld"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "afeld"
    }

  # module.uswds.github_membership.owner["rocheller123"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "rocheller123"
    }

  # module.uswds.github_membership.owner["tts-bot"] will be created
  + resource "github_membership" "owner" {
      + etag     = (known after apply)
      + id       = (known after apply)
      + role     = "admin"
      + username = "tts-bot"
    }

Plan: 36 to add, 0 to change, 0 to destroy.
```
</details>

Plan generated in [plan #74](https://github.com/18F/tts-tech-portfolio/actions/runs/1060568098)

File: how_we_work/standards.md [hunk 1]
```diff
@@ -5,17 +5,19 @@ Beyond the TTS-wide [language selection](https://engineering.18f.gov/language-se
...
+  - **Style:**
+    - Use an opinionated formatter, specifically:
+      - [Prettier](https://prettier.io/) for HTML, CSS, JavaScript, etc.
+      - [Black](https://black.readthedocs.io/en/stable/) for Python
+      - [`terraform fmt`](https://www.terraform.io/docs/commands/fmt.html) for Terraform
+    - [Soft wrap](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap) text
 - **Configuration as code:** [Terraform](https://www.terraform.io/)
-- **Continuous integration/deployment:** [CircleCI](https://circleci.com/) or [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions)
-- **Shared credentials:** [KeePassXC](https://keepassxc.org/) synced through [Google Drive File Stream](https://support.google.com/drive/answer/7329379#macos)
+- **Continuous integration/deployment:** [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions)
```
Comment (afeld) [inline how_we_work/standards.md:19]: Removed CircleCI. I don't have strong feelings about one vs. the other, but seems like we've been moving this direction over time.

File: how_we_work/standards.md [hunk 2]
```diff
@@ -38,3 +39,4 @@ Protected branches are [required](https://engineering.18f.gov/project-setup/#bra
   - In other words, it's ok to _not_ have every change to a new project reviewed.
   - This applies to code repositories (though [pull requests](https://guides.github.com/introduction/flow/)) and Google Docs (through [suggest changes](https://support.google.com/docs/answer/6033474)).
 - We try and avoid shared accounts (using one of our [Google Groups](ops_rotation.md#google-groups) as an account email) whenever possible
+- Substantive changes to How We Work should be approved by at least two members of the team.
```
Comment (afeld) [inline how_we_work/standards.md:42]: Could have sworn we had something like this written down, but couldn't find it. Open to alternate ideas.

File: how_we_work/standards.md [hunk 3]
```diff
@@ -5,17 +5,19 @@ Beyond the TTS-wide [language selection](https://engineering.18f.gov/language-se
...
-  - **Style:** Use an opinionated formatter, specifically:
-    - [Prettier](https://prettier.io/) for HTML, CSS, JavaScript, etc.
-    - [Black](https://black.readthedocs.io/en/stable/) for Python
-    - [`terraform fmt`](https://www.terraform.io/docs/commands/fmt.html) for Terraform
+  - **Style:**
+    - Use an opinionated formatter, specifically:
+      - [Prettier](https://prettier.io/) for HTML, CSS, JavaScript, etc.
+      - [Black](https://black.readthedocs.io/en/stable/) for Python
+      - [`terraform fmt`](https://www.terraform.io/docs/commands/fmt.html) for Terraform
+    - [Soft wrap](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap) text
```
Comment (afeld) [inline how_we_work/standards.md:17]: This is new.

File: how_we_work/ops_rotation.md [hunk 4]
```diff
@@ -2,9 +2,7 @@
 
 [Create an issue using the Operations Rotation template](https://github.com/18F/tts-tech-portfolio/issues/new?template=ops.md) before each Rotation to track tasks. Close once the rotation has ended.
 
-We wear a lot of different hats and it's not always clear what process to follow
-in order to address a request. In general, look for documented instructions in
-the context of the request. If you're not sure:
+We wear a lot of different hats and it's not always clear what process to follow in order to address a request. In general, look for documented instructions in the context of the request. If you're not sure:
```
Comment (adborden) [inline how_we_work/ops_rotation.md:5]: fwiw, removing the line breaks make it harder to read in plain text. Most folks configure linters/editors to warn/error when lines are longer than X characters.

File: how_we_work/standards.md (hunk 3 above)
Comment (adborden) [reply how_we_work/standards.md:17]: lol, I was just commenting about hard wrap above. Hard wrap is preferred/enforced with code linters, not sure why we want to break that convention? [PEP-8 even specifies hard wrap at 79 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) (not that I care what the max line-length is, only that it exists).

Here's an example where soft wrap with indented text causes readability problems in plain text:
Screen Shot 2021-08-05 at 5 33 21 PM

I think hard wrap is more more readable in plaintext:
```markdown
    - Default to using a repository under [github.com/18F](https://github.com/18F),
      unless there is a better place, such as contributing it to an external
      open source project.
    - Code written for one-off use can go into an issue comment or Gist.
    - Important thing is that it's not just on an individual's computer
```

File: how_we_work/standards.md [hunk 5]
```diff
@@ -5,17 +5,19 @@ Beyond the TTS-wide [language selection](https://engineering.18f.gov/language-se
...
 - **Configuration as code:** [Terraform](https://www.terraform.io/)
-- **Continuous integration/deployment:** [CircleCI](https://circleci.com/) or [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions)
-- **Shared credentials:** [KeePassXC](https://keepassxc.org/) synced through [Google Drive File Stream](https://support.google.com/drive/answer/7329379#macos)
+- **Continuous integration/deployment:** [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions)
+- **Shared credentials:** [KeePassXC](https://handbook.tts.gsa.gov/password-requirements/#1-use-a-password-manager) through Google Team Drive
```
Comment (adborden) [inline how_we_work/standards.md:20]: Should this be part of onboarding? I wasn't aware of this until today.

File: how_we_work/standards.md (hunk 2 above)
Comment (adborden) [reply how_we_work/standards.md:42]: Tough to enforce... although if we required a review from @afeld, it's on you to make sure there are two ✅ s.

**Review by adborden — COMMENTED** [review]
👍  my only hang up is the soft wrap. It seems counter to my experience where there's almost always a maximum line length enforced for code.

**Review by rocheller123 — DISMISSED** [review]

File: how_we_work/standards.md (hunk 2 above)
Comment (afeld) [reply how_we_work/standards.md:42]: True, but I'm ok with not being one of them. Let's see how this goes.

File: how_we_work/standards.md (hunk 3 above)
Comment (afeld) [reply how_we_work/standards.md:17]: Yep, I am conscientious objector to that part of PEP 8 😛 I think that readability tradeoff example is worth it, relative to the cumbersome-ness of working with hard wrapping.

File: how_we_work/standards.md (hunk 5 above)
Comment (afeld) [reply how_we_work/standards.md:20]: It should!

**Review by JJediny — DISMISSED** [review]
I think the comments don't warrant a significant change to the text. This is a living doc, as such I'm going to merge until we have a need to revisit/update

**Review by JJediny — APPROVED** [review]
LG

**Pull request merged**